### DISCUSSION TITLE
Add the possibility to hide a column with a visible prop, like the visibleMediaQuery

### DISCUSSION THREAD

**Discussion by asa75** [icflorescu/mantine-datatable#48] [Ideas]
It could be a nice solution to hide some columns, for example if the user shall not see some data...

**Comment by icflorescu** [comment]
This shouldn't be too difficult to implement, but I'd rather call the property `hidden`. Would that be ok?

**Comment by asa75** [comment]
Yes, why not.

Thank you very much.

**Comment by icflorescu** [comment]
This will land in `v1.6.3` in a few minutes.

**Comment by asa75** [comment]
Waow !

You are amazing, thank you very much, it works very well!

Have a nice day.
